VSS.md

```diff
@@ -0,0 +1,518 @@
+        - **ef\_runtime**	(integer)	The count of vectors to be examined during a query operation. The default is 10, and the max is 4096\.
+
+```
+FT._LIST
+```
+
+Lists the currently defined indexes.
+
+**RESPONSE**
+
+An array of strings which are the currently defined index names.
+
+```
+FT.SEARCH <index> <query>
+  [NOCONTENT]
+  [TIMEOUT <timeout>]
+  [PARAMS nargs <name> <value> [ <name> <value> ...]]
+  [LIMIT <offset> <num>]
+  [DIALECT <dialect>]
+```
+
+Performs a search of the specified index. The keys which match the query expression are returned.
+
+- **\<index\>** (required): This index name you want to query.  
+- **\<query\>** (required): The query string, see below for details.  
+- **NOCONTENT** (optional): When present, only the resulting key names are returned, no key values are included.  
+- **TIMEOUT \<timeout\>** (optional): Lets you set a timeout value for the search command. This must be an integer in milliSeconds.  
+- **PARAMS \<count\> \<name1\> \<value1\> \<name2\> \<value2\> ...** (optional): `count` is of the number of arguments, i.e., twice the number of value name pairs. See the query string for usage details.
+- **RETURN \<count\> \<field1\> \<field2\> ...** (options): `count` is the number of fields to return. Specifies the fields you want to retrieve from your documents, along with any aliases for the returned values. By default, all fields are returned unless the NOCONTENT option is set, in which case no fields are returned. If num is set to 0, it behaves the same as NOCONTENT.
+- **LIMIT \<offset\> \<count\>** (optional): Lets you choose a portion of the result. The first `<offset>` keys are skipped and only a maximum of `<count>` keys are included. The default is LIMIT 0 10, which returns at most 10 keys.  
+- **DIALECT \<dialect\>** (optional): Specifies your dialect. The only supported dialect is 2\.
+
+**RESPONSE**
+
+The command returns either an array if successful or an error.
+
+On success, the first entry in the response array represents the count of matching keys, followed by one array entry for each matching key. 
+Note that if  the `LIMIT` option is specified it will only control the number of returned keys and will not affect the value of the first entry.
+
+When `NOCONTENT` is specified, each entry in the response contains only the matching keyname, Otherwise, each entry includes the matching keyname, followed by an array of the returned fields.
+
+The result fields for a key consists of a set of name/value pairs. The first name/value pair is for the distance computed. The name of this pair is constructed from the vector field name prepended with "\_\_" and appended with "\_score" and the value is the computed distance. The remaining name/value pairs are the members and values of the key as controlled by the `RETURN` clause.
+
+The query string conforms to this syntax:
+
+```
+<filtering>=>[ KNN <K> @<vector_field_name> $<vector_parameter_name> <query-modifiers> ]
+```
```

Comment:
Do we have a chance to implement the Boolean query, ScriptScore Query and Re-Score Query (If the GRAPH_PQ or IVF_GRAPH_PQ index is used)

Reply:
Same answer as above.

Reply:
Thanks for your address.